Dockerfile

```diff
@@ -173,6 +173,12 @@ RUN ARCH=$(arch | sed s/aarch64/arm64/) \
     rm requirements.txt; \
     fi
 
+# Poetry 1.8.3 requires packaging version >= 22.0, see
```

Comment:
Should we say "Poetry >= 1.8.0" here as that's the version the below issue is reported against?